mimesis/enums.py

```diff
@@ -216,3 +216,14 @@ class UnitName(Enum):
     INDUCTANCE = ('henry', 'H')
     TEMPERATURE = ('Celsius', '°C')
     RADIOACTIVITY = ('becquerel', 'Bq')
+
+
+class NumType(Enum):
```

Comment:
I think that `NumTypes` sounds much better when enum contains items named in the plural.

`NumType.FLOATS` vs `NumTypes.FLOATS`. 

What do you think?